modules/tinymce/src/core/main/ts/fmt/RemoveFormat.ts

```diff
@@ -484,8 +484,9 @@ const remove = (ed: Editor, name: string, vars?: FormatVars, node?: Node | Range
       const removed = removeNodeFormat(node);
 
       // TINY-6567/TINY-7393: Include the parent if using an expanded selector format and no match was found for the current node
+      const currentNodeMatches = Arr.exists(formatList, (f) => matchName(dom, node, f));
```

Comment:
I know this is what I sent you via DMs, but this can probably be optimized a little more.
```suggestion
      const currentNodeMatches = removed || Arr.exists(formatList, (f) => matchName(dom, node, f));
```

Reply:
I really should have realized that improvement.